### ISSUE TITLE
Bug: [`eslintperfectionist/sort-switch-case`] incoherent error message `Expected "default" to come before "default".`

### ISSUE THREAD

**Issue by OlivierZal** [azat-io/eslint-plugin-perfectionist#180]
### Describe the bug

Get an incoherent eslint error message  for `eslintperfectionist/sort-switch-case`:

```
Expected "default" to come before "default".
```

### Code example

Project (for testing – remove the `eslint-disable` comment): https://github.com/OlivierZal/melcloud-api/blob/fd854ab272e3b818b0570878d9815e938481d7ec/src/facades/device_ata.ts#L48

```
#getTargetTemperatureRange(operationMode = this.setData.OperationMode): {
    max: number
    min: number
  } {
    switch (operationMode) {
      case OperationMode.auto:
        return {
          max: this.data.MaxTempAutomatic,
          min: this.data.MinTempAutomatic,
        }
      case OperationMode.cool:
      case OperationMode.dry:
        return { max: this.data.MaxTempCoolDry, min: this.data.MinTempCoolDry }
      case OperationMode.heat:
      default:
        return { max: this.data.MaxTempHeat, min: this.data.MinTempHeat }
    }
  }
```

### ESLint version

v9.7.0

### ESLint Plugin Perfectionist version

v3.0.0

### Additional comments

Here are screenshots:

<img width="563" alt="1" src="https://github.com/user-attachments/assets/3e03c43e-cd41-4e9f-adf7-e337a93650a2">
<img width="562" alt="2" src="https://github.com/user-attachments/assets/fafdaa6f-5d1d-4431-8e97-20dbb19ea1a8">

### Validations

- [X] Read the [docs](https://perfectionist.dev).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Thank you for your issue!
I was able to reproduce the problem. I'll look into fixing it.

**Comment by azat-io** [comment]
Thank you for your issue!

This problem was fixed in 742852311452be679f0f5099f9f488d5f4704542 and released in [v3.1.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v3.1.0).
